### ISSUE TITLE
Site doesn't follow W3 best practices with labeling icons

### ISSUE THREAD

**Issue by edcous** [codeths/kitcoin#47]
For example, buttons like the cog or person should have labels of some sort.

[See W3 accessibility guidelines](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0) (the main thing we need to look at is 1.1.1).

I should be able to do this myself, but opening up the issue so I don't forget (and as a place for discussion).

**Comment by edcous** [comment]
I'm busy with other stuff so I won't be able to do this right now @asportnoy

**Comment by edcous** [comment]
Maybe I'll do this sometime this weekend
